README.md

```diff
@@ -1,82 +1,209 @@
 
-    pip install https://github.com/saphetor/variant-api-client-python/archive/master.zip
+To create a virtual environment, you can follow the [user guide](https://virtualenv.pypa.io/en/stable/userguide/) or simply run:
 
-### API Documentation
+    virtualenv -p path_to/python3 venv_dir_name
+    
+Activate the virtual environment:
 
-Please visit the [api documentation](http://docs.varsome.apiary.io) to find out how to use the api and
-what values does the api provide as a response to lookup requests
+    source venv_dir_name/bin/activate
 
-### How to get an API key
 
-You generally are not required to have an api key to use the api, though the number of requests you will be able
-to issue will be throttled if they exceed a certain number.
+Finally, to use the client, either download or clone the repository from github and place the `varsome_api` 
+folder inside your project's directory, or run:
 
-Without an API key you will not be able to perform batch requests as well.
+    pip install https://github.com/saphetor/varsome-api-client-python/archive/master.zip
 
-To obtain an API key please [contact us](mailto:support@saphetor.com)
+The client will be installed within your virtual environment. Also, 2 scripts called `varsome_api_run.py` and 
+`varsome_api_annotate_vcf.py` will be available within your virtual environment's `$PATH`.
+
+### Using the scripts to directly annotate a list of variants or a VCF file
+
+#### Annotating a variant or list of variants
+Try the following query to annotate a single variant:
+
+    varsome_api_run.py -g hg19 -q 'chr7-140453136-A-T' -p add-all-data=1
+
+The script should complete without errors and display aproximately 6,700 lines of data from `dann`, `dbnsfp`, `ensembl_transcripts`, `gerp`, `gnomad_exomes`, `gnomad_exomes_coverage`, `icgc_somatic`, `ncbi_clinvar2`, `pub_med_articles`, `refseq_transcripts`, `sanger_cosmic_public`, `uniprot_variants`, `wustl_civic` etc.
+The script can also accept a text file with variants (one per line) and an optional output file to store the
+annotations in. We suggest that you don't use this script for a large number of variants, but use
+the client within your code instead.
+
+    varsome_api_run.py -g hg19 -k api_key -i variants.txt -o annotations.txt -p add-all-data=1
+    
+The command above will read variants from `variants.txt` and dump the annotations to `annotations.txt`. 
+If you don't use the `-k` parameter, the script will do as many requests as there are variants in `variants.txt`,
+and you will probably get an HTTP status code 429 (too many requests) error.
+
+#### Annotating a VCF file
+To annotate a VCF file, use: 
+
+    varsome_api_annotate_vcf.py -g hg19 -k api_key -i input.vcf -o annotated_vcf.vcf -p add-all-data=1
+    
+Notice, however, that not all available annotations will be present in the `annotated_vcf.vcf` file. Only a subset
+of the returned annotations will be available when running this script. See the "Using the client in your code" 
```

Comment:
What's missing? Why?

Reply:
I have added only a subset of annotations in the output.vcf the possible number of annotations are too many to add them all and header info in the vcf file for all of them it might take several more days with tests and changes etc.. Besides you dont know what a user might need from all of these annotations, this is why the user is referred to the section using the client,where he is instructed how to override the VCFAnnotator class with the annotations he wants. 
After all the intended use is that the end user will get the python class objects to develop an app of his own not use the run.py and annotate_vcf scripts. Even if he wants to do that he can copy the code and use his overriden vcf annotator class